.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+*  @oleksandr-pavlyk @xaleryb @ZzEeKkAa @ndgrigorian
```

Comment:
The CODEOWNERS file requires @ZzEeKkAa and I have write access, otherwise Github says it's incorrect